src/storage/mutate/DeleteEdgesProcessor.cpp

```diff
@@ -38,9 +41,22 @@ void DeleteEdgesProcessor::process(const cpp2::DeleteEdgesRequest& req) {
                                                    edgeKey.ranking,
                                                    edgeKey.dst,
                                                    std::numeric_limits<int64_t>::max());
-                doRemoveRange(spaceId, partId, start, end);
-            });
-        });
+                std::unique_ptr<kvstore::KVIterator> iter;
+                auto ret = this->kvstore_->range(spaceId, partId, start, end, &iter);
+                if (ret != kvstore::ResultCode::SUCCEEDED) {
+                    VLOG(3) << "Error! ret = " << static_cast<int32_t>(ret)
+                            << ", spaceID " << spaceId;
+                    this->onFinished();
+                    return;
```

Comment:
do we need handle LeaderChanged?

Reply:
Seems it's not a part of raft.

Reply:
> Seems it's not a part of raft.

Yes，I mixed up.

Reply:
> > Seems it's not a part of raft.
> 
> Yes，I mixed up.

Sorry, I think I mixed up it. :) This will check leader.

Reply:
Make sense. Let me handle it.